src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -1258,10 +1269,10 @@ private void startNewCronRun(
         ContinueAsNewWorkflowExecutionCommandAttributes.newBuilder()
             .setInput(startRequest.getInput())
             .setWorkflowType(startRequest.getWorkflowType())
-            .setWorkflowRunTimeoutSeconds(startRequest.getWorkflowRunTimeoutSeconds())
-            .setWorkflowTaskTimeoutSeconds(startRequest.getWorkflowTaskTimeoutSeconds())
+            .setWorkflowRunTimeout(startRequest.getWorkflowRunTimeout())
+            .setWorkflowTaskTimeout(startRequest.getWorkflowTaskTimeout())
             .setTaskQueue(startRequest.getTaskQueue())
-            .setBackoffStartIntervalInSeconds(backoffIntervalSeconds)
+            .setBackoffStartInterval(Durations.fromSeconds(backoffIntervalSeconds))
```

Comment:
can this argument come in as a java duration?

Reply:
Yes, we should remove "Seconds" from everwhere.